src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/RedisExtIntegrationTest.php

```diff
@@ -254,16 +254,17 @@ public function testSentinel(string $sentinelOptionName)
 
     public static function sentinelOptionNames(): \Generator
     {
+        yield ['sentinel'];
         yield ['redis_sentinel'];
-        yield ['sentinel_master'];
+        yield ['sentinel'];
```

Comment:
this change is wrong. `sentinel_master` should still be tested as we support it. And testing `sentinel` twice does not make sense.